app/controllers/internal/profile_fields_controller.rb

```diff
@@ -1,32 +1,50 @@
 module Internal
   class ProfileFieldsController < Internal::ApplicationController
+    ALLOWED_PARAMS = %i[
+      input_type label active placeholder_text description
+    ].freeze
     layout "internal"
 
     def index
       @profile_fields = ProfileField.all
     end
 
     def update
-      @profile_fields = ProfileField.find(params[:id])
-      @profile_fields.update!(profile_field_params)
+      profile_field = ProfileField.find(params[:id])
+      if profile_field.update(profile_field_params)
+        flash[:success] = "Profile Field updated"
+      else
+        flash[:error] = "Profile Field error: #{profile_field.errors_as_sentence}"
+      end
       redirect_to internal_profile_fields_path
     end
 
     def create
-      @profile_field = ProfileField.create(profile_field_params)
+      profile_field = ProfileField.new(profile_field_params)
+      if profile_field.save
+        flash[:success] = "Profile Field created"
+      else
+        flash[:error] = "Profile Field error: #{profile_field.errors_as_sentence}"
+      end
       redirect_to internal_profile_fields_path
     end
 
     def destroy
-      @profile_field = ProfileField.find(params[:id])
-      @profile_field.destroy
+      profile_field = ProfileField.find(params[:id])
+      if profile_field.destroy
+        flash[:success] = "Profile Field destroyed"
```

Comment:
same as the note on "update" but I would say "deleted" as "destroy" is a very technical concept here